packages/eslint-plugin/src/rules/require-await.ts

```diff
@@ -104,7 +104,8 @@ export default util.createRule({
      * function and the delegate is `true`
      */
     function markAsHasDelegateGen(node: TSESTree.YieldExpression): void {
-      if (!scopeInfo || !scopeInfo.isGen || !node.argument) {
+      /* istanbul ignore next */
```

Comment:
After I applied fixes to the existing code the coverage dropped below the threshold because it was applied to uncovered cases, what do you suggest doing here?

Reply:
We should ignore the codecov complaint. You're making the code better.

This is a downside of mandated code coverage: a change with a good refactor that reduces lines of covered code then gets punished. Pity.